packages/function-tree/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "chalk": "^1.1.3",
+    "es6-error": "^4.0.2",
```

Comment:
Okay, removed this, as not important for the test. Will create tests for errors in firebase and http